src/turbomind/utils/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ add_library(mpi_utils STATIC mpi_utils.cc)
 set_property(TARGET mpi_utils PROPERTY POSITION_INDEPENDENT_CODE  ON)
 set_property(TARGET mpi_utils PROPERTY CUDA_RESOLVE_DEVICE_SYMBOLS  ON)
 if (BUILD_MULTI_GPU)
-    target_link_libraries(mpi_utils PUBLIC mpi logger)
+    target_link_libraries(mpi_utils PUBLIC ${MPI_CXX_LIBRARIES} logger)
```

Comment:
How is this related to this PR?

Reply:
Not related. I'll remove it and pull another PR